itou/www/siae_evaluations_views/views.py

```diff
@@ -139,22 +139,24 @@ def institution_evaluated_siae_detail(
             kwargs={"evaluation_campaign_pk": evaluation_campaign.pk},
         ),
     )
+    force_accepted = evaluated_siae.final_reviewed_at and any(
+        evaluated_jobapp.state != evaluation_enums.EvaluatedJobApplicationsState.ACCEPTED
+        for evaluated_jobapp in evaluated_siae.evaluated_job_applications.all()
+    )
```

Comment:
```suggestion
    force_accepted = (
        evaluated_siae.final_reviewed_at and
        evaluated_siae.state == evaluation_enums.EvaluatedSiaeState.ACCEPTED and any(
            evaluated_jobapp.state != evaluation_enums.EvaluatedJobApplicationsState.ACCEPTED
            for evaluated_jobapp in evaluated_siae.evaluated_job_applications.all()
        )
    )
```
me paraîtrait plus juste.
Là j'ai l'impression que dans le cas d'un refus soumis par la DDETS lors de la phase contradictoire, on a `force_accepted` qui vaut `True` ?

Reply:
Et tu as raison. Ça mérite un test.

Reply:
https://github.com/betagouv/itou/pull/1970/commits/b062ed4e9f7e9026ce9a276f73a3d77e5f24d350